components/nimbus/src/tests/test_behavior.rs

```diff
@@ -1323,4 +1325,27 @@ mod event_store_tests {
 
         Ok(())
     }
+
+    fn same_week(now: DateTime<Utc>, then: DateTime<Utc>) -> bool {
+        now.weekday().num_days_from_sunday() >= then.weekday().num_days_from_sunday()
```

Comment:
I don't think `from_sunday` would work either because the default event store boots up with the beginnings of everything set to the first day of the year. I think you'd need to do something like `(<now|then>.day_of_year() / 7).floor()` to get the week w/i the context of the event store

Reply:
I think you're right. I've added `same_month`, `same_week` based on day_of_year / `ordinal0` and `same_year` tests so as not to break in January.